css/properties/counter-reset.json

```diff
@@ -49,7 +49,9 @@
             "spec_url": "https://drafts.csswg.org/css-lists-3/#valdef-counter-increment-list-item",
             "support": {
               "chrome": {
-                "version_added": "≤83"
+                "version_added": "≤83",
+                "partial_implementation": true,
+                "notes": "The <code>list-item</code> counter is <em>not</em> used when generating the default marker string on list items; see <a href='https://crbug.com/338233131'>bug 338233131</a>."
```

Comment:
What you're saying here is that the marker [in your example](https://codepen.io/glmvc/pen/dyLBEgP) does not have the correct value, right?

<img width="323" alt="image" src="https://github.com/user-attachments/assets/cfbfe51e-2991-43c2-a843-a1be14eddb94">

For reference, this is how it looks in Firefox:

<img width="323" alt="image" src="https://github.com/user-attachments/assets/72a6a9a1-b035-44e0-8188-541f7baf83be">

Reply:
Thank you, that note is much clearer! 👍